src/main/resources/springintegration/case-creation-inbound-flow.xml

```diff
@@ -97,7 +95,7 @@
   <int:channel id="caseDlqChannelXml"/>
   <int-xml:marshalling-transformer
     input-channel="caseDlqInterimChannel" marshaller="caseCreationMarshaller"
-    output-channel="caseDlqChannel" result-transformer="caseOutboundTransformer"/>
+    output-channel="caseDlqChannelXml" result-transformer="caseOutboundTransformer"/>
```

Comment:
@davidmort Here is the source of all our problems now resolved!

Reply:
😱